src/main/java/com/ericsson/ei/frontend/utils/BackEndInfoirmationControllerUtils.java

```diff
@@ -0,0 +1,197 @@
+
+    public ResponseEntity<String> handleRequestForInstances(HttpServletRequest request) {
+        try {
+            String activeInstance = null;
+            if (request.getSession().getAttribute("backEndInstanceName") != null) {
+                activeInstance = request.getSession().getAttribute("backEndInstanceName").toString();
+            }
+
+            JsonArray allAvailableInstances = backEndInstancesUtils.getBackEndsAsJsonArray();
+            allAvailableInstances = setActiveInstance(allAvailableInstances, activeInstance);
+
+            return new ResponseEntity<>(
+                    allAvailableInstances.toString(),
+                    getHeaders(), HttpStatus.OK);
+        } catch (Exception e) {
+            return new ResponseEntity<>(
+                    "[{\"name\":\"Unable to load instances\",\"host\":\"NO HOST\",\"port\":\"NO PORT\",\"path\":\"/\"}]",
+                    getHeaders(), HttpStatus.OK);
+        }
+    }
+
+    public ResponseEntity<String> handleRequestToSwitchBackEnd(HttpServletRequest request) {
+        try {
+            String selectedInstanceName = getSelectedInstanceName(request);
+            request.getSession().setAttribute("backEndInstanceName", selectedInstanceName);
+
+            return new ResponseEntity<>(
+                    "{\"message\": \"Backend instance with name '" + selectedInstanceName + "' was selected.\"}",
+                    getHeaders(), HttpStatus.OK);
+        } catch (Exception e) {
+            return new ResponseEntity<>(
+                    "Internal error" + e.getMessage(),
+                    getHeaders(), HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    public ResponseEntity<String> handleRequestToDeleteBackEnd(HttpServletRequest request) {
+        try {
+            String instanceAsString = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
+            JsonObject objectToDelete = new JsonParser().parse(instanceAsString).getAsJsonObject();
+
+            LOG.debug("Object recieved to delete: " + objectToDelete);
+
+            backEndInstancesUtils.deleteBackEnd(objectToDelete);
+            return new ResponseEntity<>(
+                    "{\"message\": \"Backend instance with name '"
+                    + objectToDelete.get("name").getAsString() + "' was deleted.\"}",
+                    getHeaders(), HttpStatus.OK);
+        } catch (Exception e) {
+            return new ResponseEntity<>(
+                    "Internal error" + e.getMessage(),
+                    getHeaders(), HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    public ResponseEntity<String> handleRequestToAddBackEnd(HttpServletRequest request) {
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
+    	LOG.error("########");
```

Comment:
Why all these Error messages? What have failed?
We will get huge Log already as it is.

Reply:
They where left after testing.